.travis.yml

```diff
@@ -21,6 +21,7 @@ install:
 
 # running tests
 script:
+  - cd /tmp
   - pytest -v --maxfail=2 --cov=jupyterhub jupyterhub/tests
```

Comment:
Thanks for investigating. The generated files haven't moved, and the cd to `/tmp` shouldn't be necessary, I just added it for debugging Travis. I, too, don't have issues running the tests locally, it is only Travis that seems to have the issue. I assumed it would be because the generated files aren't being installed in the working dir, and the installed version is what's used, not the local one. I can't figure out why the changes here would have any effect on that, since it doesn't (knowingly) affect when or how the css files are generated.